src/logic/Windows.swift

```diff
@@ -20,10 +34,21 @@ class Windows {
                let bool = sortByBooleanAttribute($0.isMinimized, $1.isMinimized) {
                 return bool
             }
+            if Preferences.sortWindows[App.app.shortcutIndex] == .alphabetical {
+                let order = ($0.title ?? "").localizedStandardCompare($1.title ?? "")
+                switch order {
+                    case .orderedAscending:
+                        return true
+                    case .orderedDescending:
+                        return false
+                    case .orderedSame:
+                        break
+                }
+            }
             return $0.lastFocusOrder < $1.lastFocusOrder
         }
     }
-
+    
```

Comment:
Maybe remove that change? It doesn't look intentional

Reply:
My IDE does that...